rpm/qm.spec

```diff
@@ -83,7 +83,7 @@ sed -i 's/^install: man all/install:/' Makefile
 
 # Set AllowedCPUs in quadlet file
 NPROC=$(nproc)
-if [[ $NPROC == 2 ]]; then
+if [[ $NPROC <= 2 ]]; then
```

Comment:
I am not sure if this will work correctly, IE Will 1 work when there is only 1 cpu?  Should it be 0?

Reply:
oh you're right, my bad. sorry